s3/pkg/service/object.go

```diff
@@ -531,6 +532,7 @@ func (s *s3Service) CopyObject(ctx context.Context, in *pb.CopyObjectRequest, ou
 		targetObject.ObjectId = oldObj.ObjectId
 	}
 	ctx = context.WithValue(ctx, dscommon.CONTEXT_KEY_SIZE, srcObject.Size)
+	log.Info("put object ....")
```

Comment:
Same here

Reply:
Download and upload speed of some backends are slow, so, add log here for trace.

Reply:
Removed.